web/client/components/misc/combobox/PagedCombobox.jsx

```diff
@@ -134,7 +134,7 @@ class PagedCombobox extends React.Component {
             busy={this.props.busy}
             data={data}
             disabled={this.props.disabled}
-            itemComponent={this.props.itemComponent}
+            itemComponent={(other) => this.props.itemComponent({textField: this.props.textField, valueField: this.props.valueField, ...other})}
```

Comment:
Make it render a component, so it works both with functional and normal components. test

Reply:
Done